lib/core/src/lib/common/services/directionality-config-factory.ts

```diff
@@ -18,6 +18,12 @@
 import { DirectionalityConfigService } from './directionality-config.service';
 
 // eslint-disable-next-line prefer-arrow/prefer-arrow-functions
```

Comment:
Probably should be before the function?

Reply:
it's a comment block, so should be valid, the eslint positioned it like that automatically